libethcore/KeyManager.h

```diff
@@ -33,6 +33,22 @@ namespace eth
 {
 class PasswordUnknown: public Exception {};
 
+enum class PasswordValidation
+{
+	Valid,
+	Empty,
+	TooShort,
+	NotEqual
+};
+
+std::string errorDescription(PasswordValidation _validation);
+
+class PasswordValidator
```

Comment:
why a memberless class and not just a function, or at the least a static method?